src/ui/utils/telemetry.ts

```diff
@@ -39,18 +39,21 @@ export function setTelemetryContext(
   userEmail: string | undefined,
   isInternal: boolean
 ) {
-  let sentryContext: Record<string, string | boolean> = { isInternal };
+  Sentry.setTag("internal", isInternal);
+  if (userId && userEmail) {
+    Sentry.setUser({ id: userId, email: userEmail });
+    Sentry.setTag("userEmail", userEmail);
+  } else {
+    Sentry.setTag("anonymous", true);
```

Comment:
Would it be worth making this more consistent by including 
```
Sentry.setTag("anonymous", false);
```
in the first `if` block too?